cinema/models.py

```diff
@@ -47,8 +55,16 @@
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
-    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
+    movie = models.ForeignKey(
+        Movie,
+        on_delete=models.CASCADE,
+        related_name="movies"
+    )
+    cinema_hall = models.ForeignKey(
+        CinemaHall,
+        on_delete=models.CASCADE,
+        related_name="movies"
```

Comment:
The `related_name` for the `cinema_hall` ForeignKey in `MovieSession` is also set to 'movies', which can cause conflicts with other models. It's advisable to use a distinct `related_name`.